### ISSUE TITLE
ScpDriver log of installation opened after every startup

### ISSUE THREAD

**Issue by firewater2** [nefarius/ScpToolkit#514]
Like the title says. Every time I start Windows I get two logs that opens itself automatically. They're identical. The reason why it's two is that I guess I had two installs of DS4Windows at a point so maybe I installed the driver twice or something?

Anyhow this is what appears:

>     SCP Driver Installer 1.2.2.175 [05-11-2016 17:42:10]
> 
>     Detected - Microsoft Windows 8 Pro (6.2.9200.0 AMD64)
>     Selected WIN8 configuration.
> 
>     INFO - ENTER: DriverPackageInstallW
>     ERROR - Could not open file C:\Users\Cristian\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\System\ScpVBus.inf.
>     INFO - RETURN: DriverPackageInstallW (0x3)
> 
>     Install Succeeded.
> 
>     -- Install Summary --
> 
>     SCP Driver Installer 1.2.2.175 [05-11-2016 17:44:13]
> 
>     Detected - Microsoft Windows 8 Pro (6.2.9200.0 AMD64)
>     Selected WIN8 configuration.
> 
>     INFO - ENTER: DriverPackageInstallW
>     ERROR - Could not open file C:\Users\Cristian\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\System\ScpVBus.inf.
>     INFO - RETURN: DriverPackageInstallW (0x3)
> 
>     Install Succeeded.
> 
>     -- Install Summary --

The controller works fine, but this is an annoyance.

**Comment by InstallGentoo11** [comment]
Same here.

**Comment by firewater2** [comment]
This doesn't happen any more to me, but I don't recall how I fixed it. I didn't reformat or anything hard like that, so I must have removed something... I think I used VoidTools Everything to search for all instances of "ScpVBus.inf", or anything alike, and removed it all. I may also have tried removing stuff on the StartUp folder, or disabling events from Task Manager > Startup. I expected the driver not to work after that, but it does!
